css/style.css

```diff
@@ -0,0 +1,1248 @@
+/*
+BASE STYLES
+*/
+
+html {
+  box-sizing: border-box;
+}
+
+*, *:before, *:after {
+  box-sizing: inherit;
+}
+
+body,
+button,
+input,
+textarea {
+  font-family: "helvetica neue", arial, sans-serif;
+  font-size: 16px;
+}
+
+body {
+  color: #2e2e2e;
+  background-color: #fff;
+  margin: 0 auto;
+  text-align: center;
+}
+
+p,
+button,
+textarea,
+input {
+  color: #fff;
+}
+
+h1 {
+  font-size: 2em;
+  font-weight: lighter;
+  line-height: 1.3;
+}
+
+h2 {
+  font-size: 2.369em;
+  font-weight: bold;
+}
+
+h3 {
+  font-size: 1.777em;
+  font-weight: bold;
+}
+
+h4 {
+  font-size: 0.9em;
+}
+
+button {
+  background-color: #fd6632;
+  border: none;
+}
+
+a {
+  text-decoration: none;
+}
+
+button:active, a:active {
+  opacity: 0.8;
+}
+
+button:hover {
+  cursor: pointer;
+}
+
+img {
+  display: block;
+  max-width: 100%;
+  height: auto;
+  opacity: 0.7;
+}
+
+input,
+textarea {
+  display: block;
+  font-weight: bold;
+  background-color: #be4c25;
+  border: none;
+}
+
+li {
+  list-style-type: none;
+}
+
+progress[value] {
+  -webkit-appearance: none;
+  appearance: none;
+  height: 1.2em;
+  width: 100%;
+}
+
+/*
+MODIFIERS
+*/
+
+.white-color {
+  color: #fff;
+}
+
+.circle {
+  border-radius: 100%;
+}
+
+/*
+ICON FONTS
+*/
+
+@font-face {
+  font-family: 'ico-fonts';
+  src:url('/css/fonts/ico-fonts.eot?ghgy45');
+  src:url('/css/fonts/ico-fonts.eot?#iefixghgy45') format('embedded-opentype'),
+    url('/css/fonts/ico-fonts.ttf?ghgy45') format('truetype'),
+    url('/css/fonts/ico-fonts.woff?ghgy45') format('woff'),
+    url('/css/fonts/ico-fonts.svg?ghgy45#ico-fonts') format('svg');
+  font-weight: normal;
+  font-style: normal;
+}
+
+[class^="icon-"], [class*=" icon-"] {
+  font-family: 'ico-fonts';
+  speak: none;
+  font-style: normal;
+  font-weight: normal;
+  font-variant: normal;
+  text-transform: none;
+  line-height: 1;
+
+
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
```

Comment:
I'm guessing this probably came from the instructions for installing the icon font, but note that this will not work on IE (and likely not at all on Windows), and is likely to give you some trouble on mobile as well.